src/primitives/pointND.cpp

```diff
@@ -0,0 +1 @@
+#include "glibby/primitives/point2D.h"
```

Comment:
if the cpp file is empty, you should just delete it so we don't have empty files floating around :)

Reply:
i realize that I left an empty general_math.cpp floating around, you can delete that as well xD